db/Pos/DB/Class.hs

```diff
@@ -29,7 +29,7 @@ import           Pos.Util.Iterator            (ListHolderT (..))
 
 -- TODO write a documentation. LensLike' is just a lens. Written using
 -- LensLike' to avoid rankntypes.
-class (MonadIO m, MonadThrow m) => MonadDB m where
+class (MonadIO m, MonadThrow m, MonadCatch m) => MonadDB m where
```

Comment:
IIRC, `MonadCatch` implies `MonadThrow`. Also I am not sure that it's good to have these constraints here.